eng/Versions.props

```diff
@@ -1,17 +1,21 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <MicrosoftFrameworkUdkPackageVersion>1.3.0-CI-25260.1001.221205-1000.0</MicrosoftFrameworkUdkPackageVersion>
     <MicrosoftWindowsCsWinRTPackageVersion>2.0.0</MicrosoftWindowsCsWinRTPackageVersion>
     <CsWinRTDependencyDotNetCoreSdkPackageVersion>6.0.301</CsWinRTDependencyDotNetCoreSdkPackageVersion>
     <CsWinRTDependencyDotNetCoreRuntimePackageVersion>6.0.6</CsWinRTDependencyDotNetCoreRuntimePackageVersion>
     <CsWinRTDependencyWindowsSdkVersionSuffixPackageVersion>27</CsWinRTDependencyWindowsSdkVersionSuffixPackageVersion>
+
     <!-- TODO: Fix DownloadDotNetCoreSdk.ps1 script so we don't need this property -->
     <CsWinRTDependencyDotNetCoreSdkLkgPackageVersion>$(CsWinRTDependencyDotNetCoreSdkPackageVersion)</CsWinRTDependencyDotNetCoreSdkLkgPackageVersion>
+
+    <!-- TODO: REMOVE This belongs in Version.Details.xml -->
+    <MicrosoftFrameworkUdkPackageVersion>1.2.3-CI-25233.1000.221026-1354.0</MicrosoftFrameworkUdkPackageVersion>
+
     <!-- ShortForm Version Fields -->
-    <CppWinRTVersion>2.0.220929.3</CppWinRTVersion>
-    <FrameworkUdkVersion>$(MicrosoftFrameworkUdkPackageVersion)</FrameworkUdkVersion>
     <BuildTasksGitVersion>1.1.1</BuildTasksGitVersion>
+    <CppWinRTVersion>2.0.220929.3</CppWinRTVersion>
+    <FrameworkUdkVersion>1.3.0-CI-25260.1001.221205-1000.0</FrameworkUdkVersion>
     <MicrosoftTaefVersion>10.58.210222006-develop</MicrosoftTaefVersion>
     <SourceLinkCommonVersion>1.1.1</SourceLinkCommonVersion>
```

Comment:
I think we should remove these from here because they are in Version.Dependencies.props

Reply:
We shouldn't have Versions.props. Let's talk offline

Reply:
Updated as discussed

See `eng\Version.Details.xml` for docs